public/content/documentation/native/controls/checkbox.md

```diff
@@ -0,0 +1,295 @@
+- Name, Role, State must be announced when focus is on the control. Announcing the label only in a separate swipe before the checkbox does not meet this requirement
+
+### Name
+- Programmatic name describes the purpose of the control.
+- If visible text label exists, the programmatic name should match the visible text label.
+    - **Note:** Setting a programmatic name while a visible text label exists may cause VoiceOver to duplicate the announcement of the name. If this happens, hide the visible text label from VoiceOver recognization.
+
+- **UIKit**
+  - You can programmatically set the visible label with `setTitle()`.
+    - The checkbox's title will overwrite the checkbox's `accessibilityLabel`.
+  - If a visible label is not applicable in this case, set the button's `accessibilityLabel` to the label of your choice.
+    - To do this in Interface Builder, set the label using the Identity Inspector
+  - To hide labels from VoiceOver programmatically, set the label's `isAccessibilityElement` property to `false`
+  - To hide labels from VoiceOver using Interface Builder, uncheck `Accessibility Enabled` in the Identity Inspector.
+- **SwiftUI**
+  - By default, the visible label of a Toggle is the accessibility label
+  - If no visible label, use view modifier `accessibilityLabel(_:)`.
+
+### Role
+- When using non-native controls (custom controls), roles will need to be manually coded.
+
+- **UIKit**
+  - Since there is no native checkbox in UIKit, implementing a custom checkbox may be necessary using `UIButton`, `UISwitch`, `UIControl`, or another class.
+    - Extend from one of the UIKit native controls above. A common control for custom checkboxes to extend from is the `UIButton`, but choose the native control that best fits your use case.
+    - Provide your own checked and unchecked checkbox images
+    - Implement action handlers for your custom checkbox
+    - Set accessibility properties accordingly
+  - If necessary, set `accessibilityTraits` to `.button`. Be sure to set the accessibility value to either "Checked" or "Unchecked" to indicate that this control behaves as a checkbox.
+  - An alternative to setting the accessibility trait to `.button` is removing and hiding the accessibility trait using `accessibilityTraits.remove(:)`. Then, append ", Checkbox" to the programmatic name
+- **SwiftUI**
+  - Use native `Toggle` view with `toggleStyle(.checkbox)` 
+
+### Groupings
+- Group visible label with checkbox, if applicable, to provide a programmatic name for the checkbox.
+- Group the units such that the label, role, and state of the checkbox is announced in a single announcement.
+
+- **UIKit**
+  1. Ensure that the child elements of the overarching view you want to group in has their `isAccessibilityElement` properties set to false.
```

Comment:
this 1 and 2 below it come through as numbered list items when they are not in current-day magentaA11Y (I think this happened in my first few component PRs too, and I'm not sure how to prevent it).